networking_generic_switch/devices/netmiko_devices/cumulus.py

```diff
@@ -87,3 +87,79 @@ class Cumulus(netmiko_devices.NetmikoSwitch):
         re.compile(r'command not found'),
         re.compile(r'is not a physical interface on this switch'),
     ]
+
+
+class CumulusNVUE(netmiko_devices.NetmikoSwitch):
+    """Built for Cumulus 5.x
+
+    Note for this switch you want config like this,
+    where secret is the password needed for sudo su:
+
+    [genericswitch:<hostname>]
+    device_type = netmiko_cumulus
+    ip = <ip>
+    username = <username>
+    password = <password>
+    secret = <password for sudo>
+    ngs_physical_networks = physnet1
+    ngs_max_connections = 1
+    ngs_port_default_vlan = 123
+    ngs_disable_inactive_ports = False
+    """
+    NETMIKO_DEVICE_TYPE = "linux"
+
+    ADD_NETWORK = [
+        'nv set bridge domain br_default vlan {segmentation_id}',
+    ]
+
+    DELETE_NETWORK = [
+        'nv unset bridge domain br_default vlan {segmentation_id}',
+    ]
+
+    PLUG_PORT_TO_NETWORK = [
+        'nv set interface {port} bridge domain br_default access '
+        '{segmentation_id}',
+    ]
+
+    DELETE_PORT = [
+        'nv unset interface {port} bridge domain br_default access '
+        '{segmentation_id}',
```

Comment:
been testing it elsewhere, and delete port is incorrect, it shouldn't have segmentation_id:

```
 read_channel: nv unset interface swp1 bridge domain br_default access 71
nv set interface swp1 bridge domain br_default access 72
nv config apply --assume-yes
Error: Invalid parameter: '71'
```

Reply:
even if I remove this part of the code, the driver is not changing the switch port vlans (cumulus 5.6), and neutron with debug is not providing any information about port binding using netmiko. Cumulus 4 works fine. Need to test it more.

Reply:
this one works - https://review.opendev.org/c/openstack/networking-generic-switch/+/907551

Reply:
remove/applied access vlan correctly:

```nv unset interface swp2 bridge domain br_default access
nv set interface swp2 bridge domain br_default access 71
nv config apply --assume-yes
nv unset interface swp2 bridge domain br_default access

nv set interface swp2 bridge domain br_default access 71
nv config apply --assume-yes```

Reply:
I've fixed up the unit tests upstream and updated the cherry pick in this PR.

Reply:
Is it time to merge it finally?